client/src/apps/code-collabo/components/AppPageStructure.tsx

```diff
@@ -20,6 +20,11 @@ export default function PageStructure({ children }: { children: ReactNode }) {
   const { thisPage, pageTitle } = getPage(pathname, urlStart, appInfo.name);
 
   const  { isMobile }  = useScreenDimensions();
+  const [isSidebarOpen, setIsSidebarOpen] = useState(true);
+
+  const toggleSidebar = () => {
+    setIsSidebarOpen(!isSidebarOpen);
+  };
```

Comment:
Can we make this logic an external function/hook? We'll be needing it in other places on the website.

Reply:
Done

Reply:
Help with the todo for filters on mobile: https://github.com/code-collabo/collabo-community-app/pull/86/files#diff-fb1c10e8b38585af4f358b5c6a960930ec17b53c2d42a7c43d5f93bb7f8b4216L65
And toggle accordingly.

Reply:
Help with the todo for filters on mobile: https://github.com/code-collabo/collabo-community-app/pull/86/files#diff-fb1c10e8b38585af4f358b5c6a960930ec17b53c2d42a7c43d5f93bb7f8b4216L65
And toggle accordingly.

> There is a filter component already, you don't have to remake another one